playbooks/katello_stable.yml

```diff
@@ -0,0 +1,4 @@
+# For roles to run after a stable Katello development image is provisioned
+- hosts: all
```

Comment:
How about something akin to `setup_user_home` or `setup_user_config` or `configure_user_home_directory` as the playbook name?